etsi_its_conversion/etsi_its_conversion/config/params.ros1.yml

```diff
@@ -1,7 +1,12 @@
 has_btp_destination_port: true
 btp_destination_port_offset: 8  # default for Cohda Wireless MK-5 exampleETSI application
 etsi_message_payload_offset: 78 # default for Cohda Wireless MK-5 exampleETSI application
-etsi_types:
+ros2udp_etsi_types:
+  - cam
+  - cam_ts
+  - cpm_ts
+  - denm
+udp2ros_etsi_types:
   - cam
```

Comment:
For `ros2udp_etsi_types` we have both `cam` and `cam_ts` within the list but not for `udp2ros_etsi_types` even if this would be a valid parameterization

Reply:
keep and mention in README as discussed